### PR TITLE
[TkAl] Update PV Validation and PV Resolution per-run submission tools 

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

The aim of this PR is to update the Tracker Alignment PV Validation and PV Resolution per run submission tools.
The list of features introduced in this PR are:
  * added the executable script: `submitPVValidationJobs.py`;
  * added possibility of configuring refitter sequence and CPE type from submission;
  * updated the example ini files with the lauching script, now in the path;
  * updated unit test to run job submitters;
  * included all modification to be compliant with both Py2 and Py3;

#### PR validation:

Relies on the existing (and augmented) unit tests, both in the regular build  `CMSSW_11_2_X_2020-10-01-2300` and in the Py3 build  `CMSSW_11_2_PY3_X_2020-10-04-2300`.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

This is not a backport, and no backport is immediately needed.

### CODE REVIEW

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by christopheralanwest** [comment]
please test

The errors seen previously seem likely to be only transient.

**Comment by mmusich** [comment]
@cmsbuild, please test

File: Alignment/OfflineValidation/test/testPVValidation_Relvals_DATA.ini [hunk 1]
```diff
@@ -24,11 +28,21 @@ bowtag: TrackerSurafceDeformations_v1_express
...
+#ttrhtype: WithAngleAndTemplate
+
 [Selection]
 applyruncontrol: True
 ptcut: 3.
 runboundary: 275657 
-lumilist: /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions16/13TeV/DCSOnly/json_DCSONLY.txt 
+# also on afs
+#lumilist: /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions16/13TeV/DCSOnly/json_DCSONLY.txt
+lumilist: /eos/cms/store/group/comm_dqm/certification/Collisions16/13TeV/DCSOnly/json_DCSONLY.txt
```
Comment (mmusich) [inline Alignment/OfflineValidation/test/testPVValidation_Relvals_DATA.ini:42]: @smuzaffar out of curiosity, the cms-bot does not see this eos partition?
I get this error in the unit tests [logs](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-b6c432/9790/unitTests.log):
```bash
Traceback (most recent call last):
  File "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-10-07-1100/bin/slc7_amd64_gcc820/submitPVValidationJobs.py", line 1029, in <module>
    main()
  File "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-10-07-1100/bin/slc7_amd64_gcc820/submitPVValidationJobs.py", line 790, in main
    if not isInJSON(run,lumilist[0]):
  File "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-10-07-1100/bin/slc7_amd64_gcc820/submitPVValidationJobs.py", line 161, in isInJSON
    with open(jsonfile, 'rb') as myJSON:
IOError: [Errno 2] No such file or directory: u'/eos/cms/store/group/comm_dqm/certification/Collisions16/13TeV/DCSOnly/json_DCSONLY.txt'
Failure running PV Validaiton run-by-run submission: status 1
status = 256
```

**Comment by smuzaffar** [comment]
@mmusich , we do not have direct access to /eos on build or dev machines. You need to open the file using xrootd

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by yuanchao** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**